ComponentKit/TransactionalDataSources/Common/CKTransactionalComponentDataSourceAppliedChanges.h

```diff
@@ -32,4 +32,6 @@
                        insertedIndexPaths:(NSSet *)insertedIndexPaths
                                  userInfo:(NSDictionary *)userInfo NS_DESIGNATED_INITIALIZER;
 
+- (NSIndexPath *)newIndexPathForPreviousIndexPath:(NSIndexPath *)indexPath;
```

Comment:
It could be, yeah! I want to write tests either way. What do you think it should be called?